tests/doc_task_test.py

```diff
@@ -461,3 +462,25 @@ class MockDbtSugarConfig:
     )
     doc_task.document_columns(doc_task.dbt_definitions)
     doc_task.dbt_definitions = {"columnA": "newDescriptionA", "columnB": "descriptionB"}
+
+
+@pytest.mark.parametrize(
+    "model_name, path_model, is_schema",
```

Comment:
what does `is_schema` mean? does it mean something like "schema exists"? if so how about renaming this to `schema_exists`?